### ISSUE TITLE
Interface not working correctly when NanoDLP password is set.

### ISSUE THREAD

**Issue by robpower** [Tibus/TouchScreen-For-NanoDLP#4]
Interface stops working if you set up a password on NanoDLP. Plates do not show up ("Undefined") and commands do not work (Z-axis, etc).
Looks like authentication needs be implemented. This should mean adding user/pass parameters to the HTTP request or using a cookie.
I'm new to node.js, but I could try to help adding it. I guess it should be in the "_request" function.

**Comment by Tibus** [comment]
Hello, never tested the authentication on NanoDLP. It could be a problem indeed. Don't have time now so feel free to add it and make a pull request ;D

**Comment by robpower** [comment]
Working on it, I already figured out part of the process. Just a question, what is the difference between src and bin folders? Should I change files in /bin?

**Comment by Tibus** [comment]
/bin folder are build from /src with gulp so you need to install node, gulp in global and run gulp work and every file in /src folder will be build in /bin when you edit a file.

**Comment by robpower** [comment]
Thanks. 
A question: which version of nodejs have you been using? I get a compile error when I run gulp, both in Windows and in Ubuntu.
In both cases I did the same: first I installed nodejs and then gulp with needed dependencies:
`npm install --global gulp-cli`
Then inside the cloned project dir:
`npm install --save-dev gulp`
`npm install gulp-plumber gulp-babel babel-core del array-includes `
When I try to run `gulp work` I get the following output:

> Using gulpfile ~/Sources/PHD/TouchScreen-For-NanoDLP/gulpfile.js
[13:47:03] Starting 'build-compile'...
[13:47:03] Finished 'build-compile' after 10 ms
[13:47:03] Starting 'build-copy'...
[13:47:03] Finished 'build-copy' after 1.92 ms
[13:47:03] Starting 'build'...
[13:47:03] Finished 'build' after 14 μs
[13:47:03] Starting 'watch'...
[13:47:03] Finished 'watch' after 33 ms
[13:47:03] Starting 'work'...
[13:47:03] Finished 'work' after 1.68 μs
[13:47:03] [Compilation Error]
[13:47:03] Plugin 0 specified in "base" was expected to return an object when invoked but returned "boolean"base0
[13:47:03] [Compilation Error]
[13:47:03] Plugin 0 specified in "base" was expected to return an object when invoked but returned "boolean"base0

Same compilation error is repeated about 20 times, I cut the output. Possibly a nodejs version incompatibility? Any other hints?
PS: module array-includes is not set as required in gulpfile.js, but if missing throws a compile error; I will also add the requirement in the pull request.
